pkg/volume/cephfs/cephfs.go

```diff
@@ -331,9 +331,8 @@ func (cephfsMounter *cephfsMounter) checkFuseMount() bool {
 	execute := cephfsMounter.plugin.host.GetExec(cephfsMounter.plugin.GetPluginName())
 	switch runtime.GOOS {
 	case "linux":
-		retBytes, err := execute.Run("/bin/ls", "/sbin/mount.fuse.ceph")
-		if err == nil && string(retBytes) == "/sbin/mount.fuse.ceph\n" {
-			glog.V(4).Infof("/sbin/mount.fuse.ceph exists, it should be fuse mount")
+		if _, err := execute.Run("test", "-x", "/sbin/mount.fuse.ceph"); err == nil {
```

Comment:
ls and test are both with coreutils. If ls is missing, so is test.

Reply:
we have testify them both.  `test` works well， but `ls` does not. We do not know why. we have some discuss before in https://github.com/kubernetes/kubernetes/pull/55992

Reply:
which linux distribution you are using?

Reply:
We are using linux distribution based on CentOS 7.3

Reply:
I suspect there is a `ls` alias somewhere in some shell profiles.

Just to be precise, use abs path `/usr/bin/test`

Reply:
thanks very much. you are right~ =)